src/test/java/EditingNotes.java

```diff
@@ -0,0 +1,40 @@
+import business_objects.Note;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import utils.androiddriver.AndroidDriverSingletone;
+import utils.dataProviders.NoteDataProvider;
+
+public class EditingNotes extends BaseTest{
+
+    @Test(dataProviderClass = NoteDataProvider.class, dataProvider = "EditNoteTitle")
+    public void editNoteTitle(Note editedNote, Note desiredNote) {
+        SoftAssert softAssert = new SoftAssert();
+        String oldTitle = editedNote.getName();
+        String newTitle = desiredNote.getName();
+        baseEmulatorPageSteps.preparationActivities();
+        softAssert.assertEquals(0, homePageSteps.getNotesCount(), "There are no notes on Hame page");
```

Comment:
Sequence of parameter in assertions should be: actial, expected

- [ ] Update perams sequence

Reply:
done